src/libraries/System.Private.CoreLib/src/System/Reflection/ConstructorInvoker.cs

```diff
@@ -2,25 +2,94 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Diagnostics;
+using System.Runtime.CompilerServices;
 
 namespace System.Reflection
 {
     internal sealed partial class ConstructorInvoker
     {
-        private readonly RuntimeConstructorInfo _constructorInfo;
-        public InvocationFlags _invocationFlags;
+        private readonly RuntimeConstructorInfo _method;
+
+#if !MONO // Temporary until Mono is updated.
+        private bool _invoked;
+        private bool _strategyDetermined;
+        private InvokerEmitUtil.InvokeFunc? _emitInvoke;
```

Comment:
`_emitInvoke` makes it sound like that this will emit the Invoke method. Would it make sense to call it `_invokeFunc` to match the delegate name?